moto/cloudformation/responses.py

```diff
@@ -1108,7 +1108,8 @@ def update_stack_instances(self):
   <StopStackSetOperationResult/>
   <ResponseMetadata>
     <RequestId>2188554a-07c6-4396-b2c5-example</RequestId>
-  </ResponseMetadata>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     </StopStackSetOperationResponse>
+  </ResponseMetadata>
+</StopStackSetOperationResponse>
```

Comment:
Bizarre that Python/boto never complained about this.. that's quite an oversight